README.md

```diff
@@ -50,52 +57,87 @@ Run the client specifying an email address you want to associate the data with
 | `IMUP_REALTIME_AUTHORIZED`         | imup API address for real-time authorized | `"https://api.imup.io/v1/auth/real-timeAuthorized"`          |
 | `IMUP_REALTIME_CONFIG`             | imup API address for reloadable config    | `"https://api.imup.io/v1/realtime/config"`                   |
 | `IMUP_DATA_LENGTH`                 | imup data length per interval             | `"15"`                                                       |
-| `LOG_TO_FILE`                      | log output to a file in the default dir   | `"false"`                                                    |
 | `PING_ADDRESS`                     | address to ping                           | `"1.1.1.1,1.0.0.1,8.8.8.8,8.8.4.4"` (CloudFlare /Google DNS) |
 | `PING_ADDRESS_INTERNAL`            | configurable gateway address              | discovered/configurable (disabled with --no-discover-gateway)|
 | `PING_DELAY`                       | time between pings in milliseconds        | `"100"`                                                      |
 | `PING_INTERVAL`                    | ping interval in seconds                  | `"60"`                                                       |
 | `PING_REQUESTS`                    | number of requests each test              | `"600"`                                                      |
-| `CONN_DELAY`                       | time between dials in milliseconds        | `"200"`                                                      |
-| `CONN_INTERVAL`                    | dialer interval in seconds                | `"60"`                                                       |
-| `CONN_REQUESTS`                    | number of requests each test              | `"300"`                                                      |
 | `SPEEDTEST_ENABLED`                | enable speed tests                        | `"false"`                                                    |
 | `SPEEDTEST_INTERVAL`               | intended to set cron frequency            | VARIABLE CURRENTLY UNUSED                                    |
 | `VERBOSITY`                        | controls log output                       | `"info"`                                                     |
 
 ## Flags
 
-```sh
+```txt
 Usage: imupClient
+  -allowlisted-ips string
+     Allowed IPs for speed tests
```

Comment:
I believe this description is outdated and allow/block listing has been extended to connectivity testing as well